src/main/scala/com/mowczare/kafka/streams/example/stream/ExampleStream.scala

```diff
@@ -35,5 +41,54 @@ object ExampleStream {
       .stream[String, InputEvent](inputTopic)
       .filter((_, _) => false)
       .to(outputTopic)
+
+
+  }
+
+  def streamTopologyHll(inputTopic: String, outputTopic: String)(builder: StreamsBuilder): Unit = {
+
+    import org.apache.kafka.streams.scala.Serdes._
+    implicit val inputEventSerde: Serde[InputEvent] = SerdeUtil.codecToSerde[InputEvent]
+    import SerdeUtil._
+    import com.mowczare.kafka.streams.pds.hashing.GenCodecHashing._
+    import org.apache.kafka.streams.scala.ImplicitConversions._
+
+    builder
+      .stream[String, InputEvent](inputTopic)
+      .groupBy { case (key, event) => event.value % 2 }
+      .hll()
+      .toStream
+      .to(outputTopic)(implicitly[Produced[Long, Hll[InputEvent]]])
+  }
+
+  def streamTopologyTheta(inputTopic: String, outputTopic: String)(builder: StreamsBuilder): Unit = {
+
+    import org.apache.kafka.streams.scala.Serdes._
+    implicit val inputEventSerde: Serde[InputEvent] = SerdeUtil.codecToSerde[InputEvent]
+    import SerdeUtil._
+    import com.mowczare.kafka.streams.pds.hashing.GenCodecHashing._
+    import org.apache.kafka.streams.scala.ImplicitConversions._
+
+    builder
+      .stream[String, InputEvent](inputTopic)
+      .groupBy { case (key, event) => event.value % 2 }
+      .theta()
+      .toStream
+      .to(outputTopic)(implicitly[Produced[Long, UpdateTheta[InputEvent]]])
+  }
+
+  def streamTopologyFrequency(capacity: Int)(inputTopic: String, outputTopic: String)(builder: StreamsBuilder): Unit = {
```

Comment:
This topology generate exception during the (de)serialization phase when called in `Launcher`.

Reply:
fixed